ui/components/app/wallet-overview/aggregated-percentage-overview.tsx

```diff
@@ -154,3 +155,83 @@ export const AggregatedPercentageOverview = () => {
     </Box>
   );
 };
+
+export const AggregatedMultichainPercentageOverview = ({
+  privacyMode = false,
+}: {
+  privacyMode?: boolean;
+}) => {
+  const locale = useSelector(getIntlLocale);
+  const currentCurrency = useSelector(getCurrentCurrency);
+  const selectedAccount = useSelector(getSelectedInternalAccount);
+  const historicalAggregatedBalances = useSelector((state) =>
+    getHistoricalMultichainAggregatedBalance(state, selectedAccount),
+  );
+
+  let color = TextColor.textAlternative;
+
+  const singleDayPercentChange = historicalAggregatedBalances.P1D.percentChange;
+  const singleDayAmountChange = historicalAggregatedBalances.P1D.amountChange;
+  const signPrefix = singleDayPercentChange >= 0 ? '+' : '-';
+
+  if (!privacyMode && isValidAmount(singleDayPercentChange)) {
+    if ((singleDayPercentChange as number) === 0) {
+      color = TextColor.textAlternative;
+    } else if ((singleDayPercentChange as number) > 0) {
+      color = TextColor.successDefault;
+    } else {
+      color = TextColor.errorDefault;
+    }
+  } else {
+    color = TextColor.textAlternative;
+  }
+
+  const localizedAmountChange = formatWithThreshold(
+    Math.abs(singleDayAmountChange),
+    0.01,
+    locale,
+    {
+      style: 'currency',
+      currency: currentCurrency,
+    },
+  );
+
+  const localizedPercentChange = formatWithThreshold(
+    Math.abs(singleDayPercentChange) / 100,
+    0.0001,
+    locale,
+    {
+      style: 'percent',
+      maximumFractionDigits: 2,
+      minimumFractionDigits: 2,
+    },
+  );
+
+  return (
+    <Box display={Display.Flex}>
+      <SensitiveText
+        variant={TextVariant.bodyMdMedium}
+        color={color}
+        data-testid="aggregated-value-change"
+        style={{ whiteSpace: 'pre' }}
+        isHidden={privacyMode}
+        ellipsis
+        length="10"
```

Comment:
nit - I haven't tested, but do we think this length is too large?
If privacy is enabled, does this add 10 asterisks? Seems like a lot for a percentage change?

Reply:
Feel free to ignore this, looks like we do this everywhere else.

Reply:
Yeah I thought it was a lot too, but just took it from the EVM equivalent of this component.